Ginger/GingerCoreCommon/UIElement/ElementLocator.cs

```diff
@@ -32,7 +32,9 @@ public class ElementLocator : RepositoryItemBase
         public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(nameof(Active)); } }
 
         private ePosition mPosition { get; set; }
-
+        /// <summary>
+        /// It is used of Friendly Locator Position 
```

Comment:
add example in comment i.e. 
/// It is used of Friendly Locator Position e.g. Left, Right, Above etc.

Reply:
Done